RecoVertex/AdaptiveVertexFinder/interface/TrackVertexArbitratration.h

```diff
@@ -2,7 +2,7 @@
 #define TrackVertexArbitration_H
```

Comment:
if you don't mind to make an extra commit, please ```git mv TrackVertexArbitratration.h TrackVertexArbitration.h``` to fix the typo in the file name